pkgs/by-name/pe/perfect_dark/package.nix

```
@@ -0,0 +1,120 @@
+{
+  romID ? "ntsc-final",
+
+  lib,
+
+  fetchFromGitHub,
+  stdenv_32bit,
+
+  pkgsi686Linux, # SDL2 and zlib
+
+  pkg-config,
+  python3,
+}:
```

```suggestion
{
  lib,
  stdenv_32bit,
  fetchFromGitHub,
  pkgsi686Linux, # SDL2 and zlib
  pkg-config,
  python3Packages,
  
  romID ? "ntsc-final",
}:
```
Nit: inputs meant to be overridable are usually at the bottom. The first three are generally `lib`, followed by `stdenv`, followed by the fetcher.